community/contributing/pr_review_guidelines.rst

```diff
@@ -0,0 +1,335 @@
+yourself if you see your team members approving the changes.
+
+But there are still more things that need to be checked before a
+PR can be merged.
+
+1. Get feedback from multiple people/teams
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Try not to merge things based on one review alone, especially
+your own. Get a second opinion from your teammates, and make
+sure all the teams have been reasonably represented by the
+reviewers. For example, if a pull request adds to the documentation,
+it's often useful to let the area maintainers check it for
+factual correctness and let documentation maintainers check it
+for formatting, style, and grammar.
+
+Make sure that the reviews and approvals were left by people
+competent in that specific engine area. It is possible that
+even a long-standing member of the Godot organization left
+a review without having the relevant expertise. As a merger
+you can overrule their approving decision or their requests
+for changes.
+
+While you shouldn't approve your own PRs, you can still merge
+them after receiving positive reviews from other maintainers.
+
+2. Get feedback from the community
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Not all areas of the engine have a lot of maintainers, so
+don't shy away from asking the involved users to help (namely,
+the reporter of the bug or the submitter of the proposal).
+Even if they cannot validate the code, they can still test the
+changes with their projects and report back the results.
+
+If you know of a contributor who has recently worked in the
+area, you can also ask them to give it a look. Consider this
+both help with the PR and an act of indoctrination into our
+team.
+
+3. Git checklist
+^^^^^^^^^^^^^^^^
+
+* **Make sure that the PR comes in one commit.**
+
+  For some cases it may be okay to have it spread across
+  several commits, but this is not the case for the majority
+  of contributions. If the pull request consists of several
+  commits, each commit must be as functional as if other
+  commits didn't exist.
+
+* **Fixes made during the review process must be squashed into
+  the main commit.**
+
+  For multi-commit PRs check that those fixes are amended in
+  the relevant commits, and are not just applied on top of
+  everything.
+
+* **Make sure that the PR has no merge conflicts.**
+
+  It must be a one click merge, and if it's not the contributor
+  needs to rebase their work from the current master.
+
+* **Check for proper commit attribution.**
+
+  This can primarily happen with new contributors, as they
+  often don't provide a correct author signature in their
+  commits. This can result in the PR being authored by
+  seemingly one person, but submitted for review by another.
+  Ultimately, it's up to them if they want it to fix, but such
+  PRs won't count towards their contributions to the project
+  and will keep them forever "New contributor" as far as our
+  project is concerned.
```

Comment:
I think we should specifically mention that this is usually the result of the email in the commit not matching with the email on GitHub.